tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -82,6 +85,31 @@ Verify NFS Volume Already Created
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error response from daemon: A volume named ${containerVolName} already exists. Choose a different volume name.
 
+Reboot VM and Verify Basic VCH Info
+    Log To Console  Rebooting VCH\n - %{VCH-NAME}
+    Reboot VM  %{VCH-NAME}
+
+    Log To Console  Getting VCH IP ...
+    ${new_vch_ip}=  Get VM IP  %{VCH-NAME}
+    Log To Console  New VCH IP is ${new_vch_ip}
+    Replace String  %{VCH-PARAMS}  %{VCH-IP}  ${new_vch_ip}
+
+    # wait for docker info to succeed
+    Wait Until Keyword Succeeds  20x  5 seconds  Run Docker Info  %{VCH-PARAMS}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  ${busybox}
+
+Verify Volume Inspect Info
+    [Arguments]  ${inspectedWhen}
+    Log To Console  \nContainer Mount Inspected ${inspectedWhen} VCH restart
+    ${rc}  ${mountInfo}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect -f '{{.Mounts}}' ${mntDataTestContainer}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${mountInfo}  ${mntTest}
+    Should Contain  ${mountInfo}  ${mntNamed}
+    Should Contain  ${mountInfo}  ${nfsNamedVolume}
```

Comment:
- [ ] Modify call to this keyword to match Docker-Util.robot's newly created keyword in this file for "Verify Volume Inspect Info"